docs/src/content/docs/ja/principles/ssh.md

```
@@ -0,0 +1,304 @@
+---
+title: SSH 接続
+description: SSH 接続の確立と管理の仕組み
+---
+
+Server Box における SSH 接続の仕組みについて解説します。
+
+## 接続フロー
+
+```
```

markdownlint-cli2 (0.20.0)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)